### ISSUE TITLE
When playing in enve it won't play the full animation

### ISSUE THREAD

**Issue by NothingAnimated** [MaurycyLiebner/enve#274]
When playing in enve it stops over and over and won't play every frame only a few frames:  
![problem](https://user-images.githubusercontent.com/69369888/127753128-77688372-4821-4509-ae7b-b652c9bada15.gif)
I don't know if im doing anything wrong.

**Comment by NothingAnimated** [comment]
I think it could be a rendering issue and maybe instead of having to render on play back you can choose wether to render on play or not render on lay (i think this is possible because audio scrubbing is there without rendering)

**Comment by NothingAnimated** [comment]
well im trying it now and it is working though i still think adding a choice would help with any issues in the future.

**Comment by irwant** [comment]
check  the scene setting, and see value on end frame

**Comment by mihadalam56** [comment]
Clean your ram or change the preview resolution

**Comment by NothingAnimated** [comment]
the problem is that it's random. Sometimes it play 80 frames sometimes the whole thing sometimes only like 6 frames.

**Comment by NothingAnimated** [comment]
the only thing that kinda helps is turning on gpu acceleration but even then not by much.

**Comment by NothingAnimated** [comment]
Yeah:
![image](https://user-images.githubusercontent.com/69369888/128416750-b460e63e-eccf-4ae2-9df0-9d231ff852c6.png)
I have 8 gigs of ram

**Comment by irwant** [comment]
> Yeah:
> ![image](https://user-images.githubusercontent.com/69369888/128416750-b460e63e-eccf-4ae2-9df0-9d231ff852c6.png)
> I have 8 gigs of ram

yes, that's is the problem. your RAM fully use by another program. Usually, I close a browser like chrome or firefox when using enve. I also using enve on the laptop with 8 GB of ram, and isn't using large files like image or video.

**Comment by NothingAnimated** [comment]
ok that fixed it thanks
